feed/history/boost_1_82_0.qbk

```diff
@@ -160,16 +160,20 @@ or GCC before 4.8.
 
 * [phrase library..[@/libs/pfr/ PFR]:]
   * Implemented the `boost::pfr::is_implicitly_reflectable` and the machinery for specializing aggregates as reflectable or not, thanks to Denis Mikhailov for the PR [github_pr pfr 111].
-  * Implemented the `BOOST_PFR_ENABLED` to macro to detect library support, thanks to Denis Mikhailov for the PR [github_pr pfr 86].
+  * Implemented `boost::pfr::get` by type of an element in the aggregate.
+  * Implemented the `BOOST_PFR_ENABLED` macro to detect library support for reflection, thanks to Denis Mikhailov for the PR [github_pr pfr 86].
   * Fixed missing terminating character, thanks to Denis Mikhailov for the PR [github_pr pfr 114].
-  * Fixed for the `boost/pfr/config.hpp` on older compilers, thanks to Denis Mikhailov for the PR [github_pr pfr 118].
+  * Fixed the `boost/pfr/config.hpp` compilation on older compilers, thanks to Denis Mikhailov for the PR [github_pr pfr 118].
   * Workaround for broken C++17 stuctured binding in old compilers, thanks to Denis Mikhailov for the PR [github_pr pfr 119].
   * Avoid -Wzero-as-null-pointer-constant warnings, thanks to Markus F.X.J. Oberhumer for the PR [github_pr pfr 117].
   * Fixed multiple typos, thanks to Denis Mikhailov for the fixes.
   * Added missing `#pragma once`, thanks to [@https://github.com/caozhanhao caozhanhao] for the PR [github_pr pfr 121].
-  * Implemented `boost::pfr::get` by type of an element in the aggregate.
   * Multiple fixes and improvements for the docs.
 
+* [phrase library..[@/libs/stacktrace/ Stacktrace]:]
```

Comment:
Above changes are just fixes for typos and reordering of bullets. Only this paragraph is new and relates to post beta changes